nbt.js

```diff
@@ -50,14 +50,36 @@ const protos = {
   littleVarint: protoVarInt
 }
 
-function writeUncompressed (value, proto = 'big') {
-  if (proto === true) proto = 'little'
-  return protos[proto].createPacketBuffer('nbt', value)
+const protosSkipChecks = {
+  big: createProto('big', { skipChecks: true }),
+  little: createProto('little', { skipChecks: true }),
+  littleVarint: createProto('littleVarint', { skipChecks: true })
```

Comment:
This seems pretty ugly. Perhaps a better way would be to pass the arguments down along with .createPacketBuffer() as opposed to creating these "instances" for every option permutation.